xclim/indices/_conversion.py

```diff
@@ -1447,33 +1447,33 @@ def potential_evapotranspiration(
         pet = pet.clip(0)
 
     elif method in ["droogersallen02", "DA02"]:
-        tasmin = convert_units_to(tasmin, "degC")
-        tasmax = convert_units_to(tasmax, "degC")
-        pr = convert_units_to(pr, "mm/month", context="hydro")
+        _tasmin = convert_units_to(tasmin, "degC")
+        _tasmax = convert_units_to(tasmax, "degC")
+        _pr = convert_units_to(pr, "mm/month", context="hydro")
```

Comment:
Are those underscores a consequence of `convert_units_to` being hard to interpret by `mypy` ? If so, I would suggest not making the change here and rather wait for a typing fix in `convert_units_to`.

Reply:
Nope, this is needed because the type signature of all these variables are `Optional[xr.DataArray]`. Assigning them into new variables is needed to ensure that they don't retain the possibilty of being interpreted as `None`.